docs/admin/deployment.rst

```diff
@@ -12,12 +12,13 @@ Buildpack
 ---------
 Pontoon uses several buildpacks in a specific order. They are (in order):
 
-1. `heroku-buildpack-apt`_ for installing Subversion.
-2. `heroku-buildpack-ssh`_ for setting up the SSH keys necessary for committing
+1. `heroku-buildpack-submodules` to fetch all related git submodules.
```

Comment:
@mathjazz I think that David Ascher had the same problems: https://bugzilla.mozilla.org/show_bug.cgi?id=1286062